backend/src/packages/chaiNNer_pytorch/pytorch/utility/convert_to_onnx.py

```diff
@@ -33,7 +33,7 @@
     ],
 )
 def convert_to_onnx_node(
-    model: PyTorchSRModel, is_fp16: int
+    model: SRModelDescriptor | RestorationModelDescriptor, is_fp16: int
 ) -> tuple[OnnxGeneric, str]:
     assert not isinstance(
         model, SCUNet
```

Comment:
Wrong.

Man, I can't even fault pyright for not finding this. Since python support inheriting from multiple classes, there could be a subclass of both `SRModelDescriptor` and `SCUNet`, so it can't assume that this `isinstance` will always be false.

Reply:
gah, good catch